build/build-image/cross/VERSION

```diff
@@ -1 +1 @@
-v1.6.3-6
+v1.6.3-7
```

Comment:
Ideally, I would have gone with a patched 1.6.3, but I'm not confident enough that it would work, since both CLs I've got are targeted at HEAD
I've already back-ported them once